docs/maui/views/MediaElement.md

```diff
@@ -152,6 +212,30 @@ By default, the media that is defined by the `Source` property doesn't immediate
 
 Platform provided media playback controls are enabled by default, and can be disabled by setting the `ShouldShowPlaybackControls` property to `false`.
 
+### Using MetaData
```

Comment:
I've renamed this class to `Metadata` in the code to use [the correct spelling](https://www.merriam-webster.com/dictionary/metadata)

```suggestion
### Using Metadata
```